src/tribol/mesh/MeshData.cpp

```diff
@@ -929,47 +741,114 @@ void MeshData::print(std::ostream& os) const
    os << "{\n";
    os << axom::fmt::format("  verts ({}) {{",num_verts);
    // positions
-   os << axom::fmt::format("\n\tx: {}", axom::fmt::join(m_positionX, m_positionX+num_verts, ", "));
-   os << axom::fmt::format("\n\ty: {}", axom::fmt::join(m_positionY, m_positionY+num_verts, ", "));
-   if(dim == 3)
+   os << axom::fmt::format("\n\tx: {}", axom::fmt::join(m_position[0].data(), m_position[0].data()+num_verts, ", "));
+   os << axom::fmt::format("\n\ty: {}", axom::fmt::join(m_position[1].data(), m_position[1].data()+num_verts, ", "));
+   if(m_dim == 3)
    {  
-      os << axom::fmt::format("\n\tz: {}", axom::fmt::join(m_positionZ, m_positionZ+num_verts, ", "));
+      os << axom::fmt::format("\n\tz: {}", axom::fmt::join(m_position[2].data(), m_position[2].data()+num_verts, ", "));
    }
-   // contact force
-   if( m_forceX != nullptr )
+   // contact response (force)
+   if( !m_response.empty() )
    {
-      os << axom::fmt::format("\n\tfx: {}", axom::fmt::join(m_forceX, m_forceX+num_verts, ", "));
-      os << axom::fmt::format("\n\tfy: {}", axom::fmt::join(m_forceY, m_forceY+num_verts, ", "));
-      if(dim == 3)
+      os << axom::fmt::format("\n\tfx: {}", axom::fmt::join(m_response[0].data(), m_response[0].data()+num_verts, ", "));
+      os << axom::fmt::format("\n\tfy: {}", axom::fmt::join(m_response[1].data(), m_response[1].data()+num_verts, ", "));
+      if(m_dim == 3)
       {  
-         os << axom::fmt::format("\n\tfz: {}", axom::fmt::join(m_forceZ, m_forceZ+num_verts, ", "));
+         os << axom::fmt::format("\n\tfz: {}", axom::fmt::join(m_response[2].data(), m_response[2].data()+num_verts, ", "));
       }
    }
    os << "\n  }";
 
    os << axom::fmt::format("\n  elems ({}) {{", num_elem);  
    
-   if( m_connectivity != nullptr )
+   if( !m_connectivity.empty() )
    {
-      os << axom::fmt::format("\n\tconnectivity: {{ {} }}", axom::fmt::join(m_connectivity, m_connectivity+(num_elem*m_numNodesPerCell), ", "));
+      os << axom::fmt::format("\n\tconnectivity: {{ {} }}", axom::fmt::join(m_connectivity.data(), m_connectivity.data()+(num_elem*numberOfNodesPerElement()), ", "));
    }
 
    // normals
-   if( m_nX != nullptr)
-   {
-      os << axom::fmt::format("\n\tnx: {}", axom::fmt::join(m_nX, m_nX+num_elem, ", "));
-      os << axom::fmt::format("\n\tny: {}", axom::fmt::join(m_nY, m_nY+num_elem, ", "));
-      if(dim == 3)
-      {  
-         os << axom::fmt::format("\n\tnz: {}", axom::fmt::join(m_nZ, m_nZ+num_elem, ", "));
-      }
-   }
-   os << "\n  }";
+  //  if( !m_n.empty() )
```

Comment:
Is there a reason the normals are now commented out? Looks like they were not commented out in the original. Please verify.

Reply:
Needed to be updated for the new 2D array storage.  Fixed now